pom.xml

```diff
@@ -297,6 +313,17 @@
     </resources>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <configuration>
+            <skip>true</skip>
+          </configuration>
+        </plugin>
```

Comment:
This is to avoid
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:javadoc (default) on project nexus-jenkins-plugin: An error has occurred in JavaDocs report generation: 
[ERROR] Exit code: 1 - C:\sonatype\jenkins-nexus-platform-plugin\src\main\java\org\sonatype\nexus\ci\iq\ManualApplication.java:20: error: cannot find symbol
[ERROR]     extends IqApplication
[ERROR]             ^
[ERROR]   symbol: class IqApplication
...
```
Not sure how to fix it (instead of just avoiding it) but I don't believe we use the Javadoc